### PR TITLE
Cool new feature

### PR DESCRIPTION
**Pull request by ladanv**
Check it out! It's really cool!

### CODE REVIEW

File: doc.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Hello World!
 
-This is the line for editing. Stranger, please edit this line and create a git request!
\ No newline at end of file
+This is the line for editing. Stranger, please edit this line and create a git request!
+
+File was changed by ... Vitaliy L.
```
Comment (IvanStankov) [inline doc.txt]: Who are you?

File: doc.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
 Hello World!
 
-This is the line for editing. Stranger, please edit this line and create a git request!
\ No newline at end of file
+This is the line for editing. Stranger, please edit this line and create a git request!
```
Comment (IvanStankov) [inline doc.txt]: WTF here?

File: doc.txt (hunk 1 above)
Comment (cyclion) [inline doc.txt]: HEHE

Comment (ladanv) [inline doc.txt]: We haven't met before

Comment (cyclion) [inline doc.txt]: Brrrrr

File: doc.txt [hunk 3]
```diff
@@ -1,3 +1,7 @@
...
+
+File was changed by ... Vitaliy L. 
+
+Changed according review.
```
Comment (IvanStankov) [inline doc.txt]: Accepted

**Comment by IvanStankov** [comment]
Closed with smth

**Pull request merged**